src/features/block-if-missing-labels.ts

```diff
@@ -7,31 +7,32 @@ import { intersection } from "lodash";
 import { getLogger } from "../utils/logger";
 import { formattedRepoName } from "../utils/pr-context";
 
-const logger = getLogger("fail-if-no-release-label");
+const logger = getLogger("block-if-missing-labels");
 
 const getConfigLabelPairs = (labels: Config["labels"]): [string, string][] =>
   Object.entries(labels).map(([labelKey, label]) => [
     labelKey,
     typeof label === "string" ? label : label.name ? label.name : labelKey,
   ]);
 
-export default class FailIfMissingLabels extends PullRequestPlugin {
-  public name = "FailIfNoReleaseLabels";
+export class BlockIfMissingLabels extends PullRequestPlugin {
+  public name = "BlockIfMissingLabels";
   private failed = false;
 
   public apply(prHooks: Hooks["pr"]) {
-    prHooks.shouldSkipAllProcessing.tapPromise(this.name, this.isMissingRequiredLabels.bind(this));
-    prHooks.modifySkipStatus.tapPromise(this.name, this.setSkipStatus.bind(this));
+    logger.debug(`Applying hooks for ${this.name}`);
+    prHooks.process.tapPromise(this.name, this.isMissingRequiredLabels.bind(this));
+    prHooks.modifyCompleteStatus.tapPromise(this.name, this.setStatus.bind(this));
   }
 
-  private async setSkipStatus(status: Status, context: PRContext): Promise<Status> {
+  private async setStatus(status: Status, context: PRContext): Promise<Status> {
     if (this.failed) {
       logger.info(`${formattedRepoName(context)} PR #${context.payload.number} missing required version labels`, {
         url: context.url,
       });
       return {
-        state: "failure",
-        description: "Missing version or skip-release labels",
+        state: "pending",
```

Comment:
I feel like `pending` is a better representation of the state of the pull request at the point when labels need to be added. 

Failed makes it feel like something was done wrong and a change to code is needed. Pending just  feels like it's waiting for something (like labels in this case). 

What do you think @hipstersmoothie?